00_Base/src/services/credentials.service.ts

```diff
@@ -474,6 +474,85 @@ export class CredentialsService {
     return updatedClientInformation;
   }
 
+  async deleteTenant(tenantId: string): Promise<void> {
+    const cpoTenant = await this.cpoTenantRepository.readOnlyOneByQuery({
+      where: {
+        id: tenantId,
+      },
+    });
+    if (!cpoTenant) {
+      throw new NotFoundError('CpoTenant not found');
+    }
+
+    const transaction =
```

Comment:
This is missing the association with the query.

Reply:
adjusted 👍 🙇